packages/picasso/src/QuillEditor/hooks/useQuillInstance/useQuillInstance.tsx

```diff
@@ -6,22 +6,48 @@ import {
   useTypographyClasses,
   makeHeaderFormat,
   makeBoldFormat,
+  makeLinkFormat,
 } from '../../formats'
+import type { EditorPlugin } from '../../types'
 
 export type EditorOptionsType = {
   id: string
   placeholder?: string
+  plugins?: EditorPlugin[]
 }
 
-export const getModules = (): QuillOptionsStatic['modules'] => {
+export const getModules = (
+  plugins: EditorOptionsType['plugins']
+): QuillOptionsStatic['modules'] => {
+  const allowLinks = plugins?.includes('link')
+
+  const allowedTags = [
+    'b',
+    'strong',
+    'i',
+    'em',
+    'p',
+    'br',
+    'ul',
+    'ol',
+    'li',
+    'h3',
+  ]
+  const allowedAttributes = []
+
+  if (allowLinks) {
+    allowedTags.push('a')
+    allowedAttributes.push('href')
+  }
```

Comment:
We could simplify here and inline this `if` condition, wdyt?

```suggestion
    'h3',
    ...allowLinks ? ['a', 'href'] : []
  ]
  const allowedAttributes = []
```